src/azure-cli/azure/cli/command_modules/databoxedge/generated/commands.py

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from ..generated._client_factory import cf_device
+    databoxedge_device = CliCommandType(
+        operations_tmpl='azure.mgmt.databoxedge.operations._devices_operations#DevicesOperations.{}',
+        client_factory=cf_device)
+    with self.command_group('databoxedge device', databoxedge_device, client_factory=cf_device) as g:
```

Comment:
Add resource type and min_api for commands or arguments to make sure unsupported commands hidden in unsupported environment

Reply:
modified